### DISCUSSION TITLE
GPU vs CPU performance

### DISCUSSION THREAD

**Discussion by sohanasarah** [ourownstory/neural_prophet#1195] [General]
I am training neural-prophet with a large dataset. I am trying to use accelerator: gpu feature but I noticed that training performance is better on cpu than gpu. I tried to increase batch size and decrease epoch still I get the same result. Am I missing something? Below is the parameter setup that I used:

params = dict(
    weekly_seasonality = False,
    daily_seasonality = False,
    seasonality_mode = 'multiplicative',
    epochs=500,
    learning_rate=0.05,   
    accelerator = "gpu"
)

**Comment by Kevin-Chen0** [comment]
^ @karl-richter

**Comment by lilGao16** [comment]
me too,really wonder if it's normal or there is other parameters i can adjust
